base_location_geonames_import/models/country.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Nicolas Bessi, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class ResCountryState(models.Model):
+
+    _inherit = 'res.country'
+
+    geonames_state_name_column = fields.Integer(
+        u'Geoname State Name Column',
```

Comment:
No need of `u'..'`, as it's not an unicode character and with Python 3 this will be needed to be changed again